lib/alp_workloads/kvm_workload_utils.pm

```diff
@@ -162,6 +173,7 @@ sub setup_services_within_kvm_container {
 }
 
 sub setup_kvm_container_from_scratch {
+    set_host_bridge_interface_with_host;
```

Comment:
This function seems more related with alp host setup, not kvm container utility function. Suggest to call this function in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/virt_autotest/prepare_alp_host.pm#L25.

Reply:
Besides, when the bridge setup fails, what logs shall we add in post_fail_hook?

Reply:
Good point! let me add check point here.